### PR TITLE
Moved OrbitVersion into OrbitBase

### PR DESCRIPTION
**Pull request by antonrohr**
OrbitVersion is used on the Client and Service, hence it lived in OrbitCore. This change moves it from OrbitCore to OrbitBase to be able to untie OrbitService from OrbitCore.

### CODE REVIEW

**Comment by dimitry-** [comment]
I am also not sure if this is generic enough to be in OrbitBase

**Comment by antonrohr** [comment]
I was also a little hesitant whether things like OrbitVersion should go into OrbitBase. I don't see any other option besides creating a new target for things that are used by OrbitService and the client side. I decided thats not worth it for OrbitVersion, which is pretty lightweight. wdyt?

**Comment by antonrohr** [comment]
Regarding the tests: I am unsure how to test the actual interesting part, which is whether the values that version.cmake computes are correct. If there are any ideas how to do that, I am all ears. Otherwise I would refer figuring that out to different PR.

**Comment by dimitry-** [comment]
Maybe create a separate component for OrbitVersion?

(I'd say anything that wants to go to OrbitBase has to have a unit-test)

**Comment by beckerhe** [comment]
Ideally the OrbitVersion-stuff should live close to the respective `main.cpp`s. If not a version change will trigger a full rebuild. Or at-least a full re-link.

I would also consider duplicating the code and having it in both modules (OrbitQt and OrbitService). I know that's far from ideal, but OrbitQt has also another version-info-thing which could be merged with that one. So there will be two versioned files - one in OrbitService and a different one in OrbitQt. Sounds like an okay-ish solution to me.

File: OrbitVersion/OrbitVersion.cpp.in [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include "OrbitBase/OrbitVersion.h"
+#include "OrbitVersion.h"
```
Comment (dimitry-) [inline OrbitVersion/OrbitVersion.cpp.in]: The header file should probably go to include/ dir - it is public

**Comment by antonrohr** [comment]
I decided to create a separate component. 
I will change the commit message to something meaningful when I do the rebase.

File: OrbitVersion/OrbitVersion.cpp.in (hunk 1 above)
Comment (antonrohr) [reply OrbitVersion/OrbitVersion.cpp.in]: done

**Pull request merged**